src/pages/settings/tabs/UserManagementSettings.tsx

```diff
@@ -27,6 +27,15 @@ import {
 } from '@/core/rtkQuery/plexApi';
 import { useGetAniDBTagsQuery } from '@/core/rtkQuery/splitV3Api/tagsApi';
 
+function fileToDataUrl(file: File): Promise<string | never> {
```

Comment:
```suggestion
const fileToDataUrl = (file: File): Promise<string | never> => {
```

Reply:
Also do we need to specify the return type?
It should be automatically inferred